monero-outreach-docs/translations/zh-hans/monero_resources/monero_resources_zh-hans.md

```diff
@@ -0,0 +1 @@
+Translated from https://www.monerooutreach.org/resources.php

Monero Resources
门罗币资源

Monero has a large and active community with many guides and resources available for newcomers and experts alike. This isn't a comprehensive list, but a collection of trusted sources to get started with Monero or to keep going with Monero. If you have suggestions for this section please let us know!

门罗币拥有一个庞大而活跃的社区，有许多指南和资源可供新手和专家使用。这虽然不是一个全面的列表，却是一个值得信赖的资源集合，以供你开始或者继续使用门罗币。如果你对此部分有任何建议，请让告知我们!

GET STARTED
开始

MINING & BUYING
挖矿和购买

JOIN THE COMMUNITY
加入社区

MONERO TECH
门罗币技术

```

Comment:
- Move to folder _/zh-hans/website-sections_
- Change files name to "_**resources_zh-hans.md**_"

Reply:
Done.